x/staking/keeper/genesis.go

```diff
@@ -195,18 +197,18 @@ func (k Keeper) InitGenesis(ctx context.Context, data *types.GenesisState) (res
 
 			update := validator.ABCIValidatorUpdate(k.PowerReduction(ctx))
```

Comment:
i think we can remove this section of code as well. seems we are going to a abciValidatorupdate in order to go back to a different type.

Reply:
I do not konw about this, who can helps me ? can I just delete it directly?

Reply:
basically you can call this directly in the struct: https://github.com/cosmos/cosmos-sdk/blob/d54335e1f98b224815f07b3aa8ae1e0e7ee3456d/x/staking/types/validator.go#L262

Reply:
copying that code which is mentioned. we may be able to drop the comet dependency in staking for validatorupdate entirely

Reply:
But this is another issue, can we open a separate issue for it? @julienrbrt @tac0turtle

Reply:
Imho, it's the same thing. Otherwise this PR only moves the logic a bit up which doesn't really have any advantage.

Reply:
👌, I will do it soon

Reply:
ok I will do it